host/sgx/enter.c

```diff
@@ -279,8 +313,8 @@ void oe_enter_sim(
     void* host_fs = oe_get_fs_register_base();
     void* host_gs = oe_get_gs_register_base();
     sgx_tcs_t* sgx_tcs = (sgx_tcs_t*)tcs;
-    oe_ecall_context_t ecall_context = {{0}};
-    _setup_ecall_context(&ecall_context);
+    oe_ecall_context_t ecall_context = {0};
+    oe_setup_ecall_context(&ecall_context);
```

Comment:
keep note to myself: need to check if this affects debugging in simulation mode

Reply:
Note: We might have to double-check; we don't seem to be using __morestack for simulation mode's oe_enter. I;m not sure if this was by design or an omission.

Reply:
Updated. Now simulation mode also uses the same oe_enter entry point.